controllers/assets/redis-simple-promise-updated.yaml

```diff
@@ -3,6 +3,7 @@ kind: Promise
 metadata:
   name: redis
   labels:
+    kratix.io/promise-version: v1.2.0
```

Comment:
```suggestion
    kratix.io/promise/version: v1.2.0
```

thoughts?

Reply:
dont find it better enough to change the code, but also wouldn't mind if someone wants to update it :D